### PR TITLE
Create Tailwind CSS extension

### PR DESCRIPTION
**Pull request by vimtor**
## Description

A basic [Tailwind CSS](https://tailwindcss.com) extension that lets you search quickly thought the documentation. In the future it could do things like displaying CSS classes, transforming CSS to Tailwind classes, searching components in [Tailwind UI](https://tailwindui.com), etc. That's why I named it like `Tailwind CSS` and not `Search Tailwind CSS Documentation`, but maybe this goes against the extension guidelines, so let me know 😄 .

Also [tailwindcss.com](https://tailwindcss.com) does not offer a query string search, that's why I made the extension instead of a simple command. The big object with the documentation was made on purpose instead scrapping the website since that can be really prone to errors. I would have love to use the `sitemap.xml` but I did not find it in their page.

## Type of change

- New extension

## Screencast

https://user-images.githubusercontent.com/36263538/138591476-46907778-f5e6-46ec-811b-b4e727b30312.mov

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I checked that extension's icon has size of `512x512`

### CODE REVIEW

**Comment by PitNikola** [comment]
What's the state of this extension? @vimtor, you're gonna look into using Algolia key?

**Comment by vimtor** [comment]
> What's the state of this extension? @vimtor, you're gonna look into using Algolia key?

I will prefer not to until I have explicit permission from them to be honest.

**Comment by vimtor** [comment]
I think we could follow what @thomaspaulmann said, and leave it for future versions if it becomes a problem :D

**Comment by pernielsentikaer** [comment]
It makes sense @vimtor, it would be rather cool to have it - lets chase that after first release :)

File: extensions/tailwindcss/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  "name": "tailwindcss",
+  "title": "Tailwind CSS",
+  "description": "Utilities for Tailwind CSS",
+  "icon": "tailwind-icon.png",
+  "author": "vimtor",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "search-documentation",
+      "title": "Search Tailwind CSS Documentation",
```
Comment (pernielsentikaer) [inline extensions/tailwindcss/package.json:11]:
```suggestion
      "title": "Search CSS Documentation",
      "subtitle": "Tailwind",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Just found a quick correction and then we're ready to launch it 🚀

File: extensions/tailwindcss/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "tailwindcss",
+  "title": "Tailwind CSS",
+  "description": "Utilities for Tailwind CSS",
```
Comment (PitNikola) [inline extensions/tailwindcss/package.json:4]: Considering that at the moment this extension is only about documentation, how about updating the title and description to be more specific.
title -> "Tailwind CSS Documentation"
description -> "Quickly search Tailwind CSS documentation and open it in the browser"?

Later you can change title to "Tailwind CSS" once you add more functionality

**Comment by vimtor** [comment]
@PitNikola @pernielsentikaer I added both of your comments :)

**Comment by pernielsentikaer** [comment]
Awesome @vimtor 💪

**Pull request merged**